railties/test/generators/app_generator_test.rb

```diff
@@ -695,6 +695,24 @@ def test_kamal_storage_volume_is_skipped_if_required
     end
   end
 
+  def test_gitignore_appends__storage_entries_when_active_storage_is_skippped_but_database_is_sqlite
```

Comment:
```suggestion
  def test_gitignore_appends_storage_entries_when_active_storage_is_skippped
```
We don't need to describe "but_database_is_sqlite" because that is the default, and we can assume it.

Thanks!